### ISSUE TITLE
train process issue

### ISSUE THREAD

**Issue by lingcon01** [microsoft/AI2BMD#11]
Dear author,
     Thanks for your sharing. I followed your settings to train VisNet on the Aspirin dataset, but it encountered NaN values at the 89th epoch, and the training was terminated. The energy MAE and force MAE on the test set were 0.225 and 0.453, respectively. Could you please advise on how to resolve this issue?

![Image](https://github.com/user-attachments/assets/7329c0b4-afbe-41fa-a208-3b6747e8e744)

**Comment by ElwynWang** [comment]
Such problems may be raised by various reasons. Strictly following the hyperparameters in the repo, the training process should work smoothly. I don't the specific environment and operations during your model training, but maybe you can try small lr, smaller models and longer warm up process.
